docs/devices/cloud-protocol/ncellmeas.schema.json

```
@@ -0,0 +1,110 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/devices/cloud-protocol/ncellmeas.schema.json",
+  "title": "Asset Tracker v2 neighboring cell measurement report ",
+  "description": "Describes the format which is used by the device to report neighboring cell measurements, which is produced by the %NCELLMEAS AT command. See https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/ncellmeas.html",
```

```suggestion
  "description": "Describes the format which is used by the device to report neighboring cell measurements. The report is produced by the %NCELLMEAS AT command. See https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/ncellmeas.html",
```
I believe this is correct. As it was written, the AT command produced the format, not the report.

---

This could also use the wording "The report is generated by [...}", but produces is acceptable.

---

Technically it produces a report of measurements.